File: web3rpc/rpc-specs/code-samples/python/debug/profiling/cpuProfile.py
import json
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

fileName = "block.profile"
seconds = 10

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.cpu_profile(fileName, seconds)

print(json.loads(debug_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/debug/profilling/cpuProfile.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const file = "block.profile";
    const seconds = 10;

    sdk.debug.cpuProfile(file, seconds, {}, (err, data, response) => {
        console.log(data);
    });
}
)()